task-2/src/components/AddMovieForm/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { Dropdown } from '../Dropdown';
+import PropTypes from 'prop-types';
```

Comment:
Consider grouping imports of the same level together (./oneThing and ../anotherThing are different levels) and separating each level with an empty line. Additionally, order imports top-down from most-global to most-local.
E.g. https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

If you've got time, consider adding this plugin to your project to automate that.

Reply:
- Imports have been reorganised to match the correct convention.